packages/components/pie-card-container/src/index.ts

```diff
@@ -1,16 +1,63 @@
-import { LitElement, html, unsafeCSS } from 'lit';
-
+import {
+    html, LitElement, unsafeCSS, nothing,
+} from 'lit';
+import { property } from 'lit/decorators.js';
+import { validPropertyValues } from '@justeattakeaway/pie-webc-core';
 import styles from './card-container.scss?inline';
-import { CardContainerProps } from './defs';
+import { CardContainerProps, type AriaProps, variants } from './defs';
 
 // Valid values available to consumers
 export * from './defs';
 
 const componentSelector = 'pie-card-container';
 
 export class PieCardContainer extends LitElement implements CardContainerProps {
+    @property()
+    @validPropertyValues(componentSelector, variants, 'default')
+    public variant: CardContainerProps['variant'] = 'default';
+
+    @property({ type: String, reflect: true })
+    public href?: string;
+
+    @property({ type: String, reflect: true })
+    public target?: string;
+
+    @property({ type: String, reflect: true })
+    public rel?: string;
+
+    @property({ type: Boolean })
+    public disabled = false;
+
+    @property({ type: Object })
+    public aria!: AriaProps;
+
     render () {
-        return html`<h1 data-test-id="pie-card-container">Hello world!</h1>`;
+        const {
+            href,
+            target,
+            rel,
+            disabled,
+            variant,
+        } = this;
+
+        return html`
+                <div
+                    variant=${variant}
+                    class="c-card-container"
+                    data-test-id="pie-card-container"
+                    ?disabled=${disabled}>
+                    <a
+                        data-test-id="pie-card-container-link"
+                        href=${href || ''}
```

Comment:
should not this be defaulted to nothing if not passed?
```suggestion
                        href=${href || nothing}
```

Reply:
I wish it could but TS does not allow it!

<img width="763" alt="Screenshot 2023-09-05 at 08 16 00" src="https://github.com/justeattakeaway/pie/assets/16766185/91790b5e-4e04-4cd2-bdfb-bdbbe8781d54">

Reply:
aha I see, I thought the anchor was our component and was wondering why you can't. 
I believe we can replace it later when  we conditionally render the anchor / div